disassemblers/ofrak_angr/requirements.txt

```diff
@@ -1,2 +1,2 @@
-angr==9.2.93
+angr>=9.2.93
```

Comment:
I get that `angr` changes often, but that's exactly why we pin versions. I think we should bump it to a better version, but using any version at all seems irresponsible given how often they push breaking changes.